include/common.sh

```diff
@@ -47,7 +47,7 @@ debug_mode
 export MAGICDNS="${MAGICDNS:-nip.io}"
 export ENABLE_EIRINI="${ENABLE_EIRINI:-true}"
 export EMBEDDED_UAA="${EMBEDDED_UAA:-false}"
-export KIND_APIVERSION="${KIND_APIVERSION:-kind.sigs.k8s.io/v1alpha2}"
+export KIND_APIVERSION="${KIND_APIVERSION:-kind.sigs.k8s.io/v1alpha3}"
```

Comment:
is v1alpha4 since 0.6.0

Reply:
3 seems to be backward compatible so it is easier to downgrade Kind to 0.2.1 for example by the environment variable. Is there an advantage to switch this to v4 by default?

Reply:
We wanted to keep compatibility with 0.2.1 as well with this change, as some other automation scripts are depending on that version (still)

Reply:
we should keep compatibility with the APIs.
The new API introduced some minor changes, moe the API namespace *.k8s.io  into x-k8s.io and the ability to patch kubeadm per node and containerd.

Reply:
seems that you don't need it, was mostly a nit :sweat_smile: